src/main/java/bio/terra/cli/command/config/get/Format.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cli.command.config.get;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.serialization.userfacing.UFFormatConfig;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the fourth-level "terra config get format" command. */
+@Command(name = "format", description = "Output format option.")
+public class Format extends BaseCommand {
+  @CommandLine.Mixin bio.terra.cli.command.shared.options.Format formatOption;
+
+  @Override
+  protected void execute() {
+    UFFormatConfig formatOptionConfig =
+        new UFFormatConfig.Builder().formatOption(Context.getConfig().getFormat()).build();
+    formatOption.printReturnValue(formatOptionConfig, Format::printText);
+  }
+
+  public static void printText(UFFormatConfig returnValue) {
```

Comment:
Not sure whether this matters, but it seems like all of the other enum-driven `terra set config ...` commands seem to use all caps for enum-values.  Not sure it's worth adding an additional enum that maps to the existing (lower case) `FormatOptions` enum, but something to consider for consistency...

```
logging level: OFF, ERROR, WARN, INFO, DEBUG, TRACE, ALL
Browser launch mode: MANUAL, AUTO.
App launch mode: DOCKER_CONTAINER, LOCAL_PROCESS.
```

Reply:
The easiest fix would be to change the enum to be uppercase. I don't think any legacy artifacts depend on the precise spelling.

Reply:
Actually, that breaks people who expect to do `--format=json` out of habit. Instead, @marikomedlock recommends adding the fixup of that enum to PF-1075.

Reply:
waiting for PF-1075 to merge so I can upcase the enum again.